WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -83,26 +81,23 @@ class HomePagePickerFragment : Fragment() {
             setupUi()
             setupViewModel()
             setupActionListeners()
-            previewModeSelectorPopup = PreviewModeSelectorPopup(
-                    requireActivity(),
-                    homePagePickerTitlebar.previewTypeSelectorButton
-            )
         }
     }
 
     private fun HomePagePickerFragmentBinding.setupUi() {
         homePagePickerTitlebar.title.isInvisible = !displayUtils.isPhoneLandscape()
-        modalLayoutPickerHeaderSection.modalLayoutPickerTitleRow?.header?.setText(R.string.hpp_title)
-        modalLayoutPickerHeaderSection.modalLayoutPickerSubtitleRow?.description?.setText(R.string.hpp_subtitle)
+        with(modalLayoutPickerHeaderSection) {
+            modalLayoutPickerTitleRow?.header?.setText(R.string.hpp_title)
```

Comment:
I think we should also remove the `hpp_subtitle` string resource. Could be actually the reason the CI failed since it's now an unused resource, didn't check yet the CI though just searched in code and I can confirm this resource is nowhere else except in `strings.xml` files.

Reply:
I opened a suggestion PR for removing the unused resource: https://github.com/wordpress-mobile/WordPress-Android/pull/16419

Reply:
I left this in because of this:

> and assess if it should become an accessibility hint

but actually, we don't have a place for that at the moment (it wouldn't go on the `TextView` itself, as those should _not_ have a `contentDescription`, though it could go on a button later in the flow. I think it's fine to remove it for now, since we can always just add it back (and it probably deserves a new name, and possibly adjusted copy too if we do reuse it). Thanks for the suggestion PR, I'll merge it here. :+1: